### PR TITLE
[add] tolerations field to istiod

### PR DESCRIPTION
**Pull request by ymktmk**
**Please provide a description of this PR:**

This is a pull request created from this comment.
- https://github.com/istio/istio/pull/47639#discussion_r1377295460

### CODE REVIEW

**Comment by ymktmk** [comment]
@hanxiaop 
Do I need release notes?

File: manifests/charts/istio-control/istio-discovery/values.yaml [hunk 1]
```diff
@@ -34,6 +34,8 @@ pilot:
 
   affinity: {}
 
+  tolerations: []
```
Comment (hanxiaop) [inline manifests/charts/istio-control/istio-discovery/values.yaml:37]: Still need to run make gen to generate operator related files.

**Comment by hanxiaop** [comment]
> @hanxiaop Do I need release notes?

I think we do.

**Review by hanxiaop — APPROVED** [review]

**Pull request merged**